lib/presentation/widgets/input_field_widget.dart

```diff
@@ -7,20 +7,23 @@ class InputFieldWidget extends StatelessWidget {
     required this.labelText,
     this.validator,
     this.controller,
+    required this.obscureText,
+    this.suffixIcon,
   }) : super(key: key);
 
   final String labelText;
   final SingleRegistrationValidatorFunction? validator;
   final TextEditingController? controller;
+  final bool obscureText;
```

Comment:
Make it false by default and remove false from all fields, when you passing false directly